Update runtime to 49

yakushabb opened the pull request:
- Drop already provided ffmpeg extension
- Drop already provided webp-pixbuf-loader

TingPing commented:
I don’t like just disabling plugins as a solution. The issues should be reported.

yakushabb commented:
> I don’t like just disabling plugins as a solution. The issues should be reported.

Reported to the upstream: https://gitlab.gnome.org/GNOME/eog/-/issues/336

yakushabb committed:
Update runtime to 49

- Drop already provided ffmpeg extension
- Drop already provided webp-pixbuf-loader

yakushabb commented:
@TingPing Also ready for review. All basic changes.

TingPing reviewed (approved):
lgtm, needs rebased
